.github/workflows/ci.yml

```diff
@@ -62,6 +62,7 @@ jobs:
     permissions:
       contents: read
     strategy:
+      fail-fast: true
```

Comment:
For `otherjdks` below we have `fail-fast: false`.
Is it possible for `remainder` to only cancel `otherjdks`?
I think it is good to have all remainder tasks run, even if one of them fails, to be able to see whether just that one task fails or maybe others fail, too.
But if any of the `remainder` tasks fails, we don't need to run any `otherjdks` tasks - one should first fix the failures revealed in JDK 21.
Does that policy make sense? Can it be expressed here?

Reply:
The policy sounds good to me. Let me think about how to implement it.

Reply:
@wmdietl I have looked around online and github workflow syntax. I did not find a nice solution for this policy: one job depends on another job's running status and both jobs are running parallel. Do you have any idea how to achieve this purpose?

Reply:
make everything fail fast.